packages/ERTP/test/unitTests/mathHelpers/test-natMathHelpers.js

```diff
@@ -61,65 +48,235 @@ test('natMathHelpers', t => {
   );
   t.throws(
     () =>
-      coerce(
-        harden({ brand: { getAllegedName: () => 'somename' }, value: 4n }),
+      m.coerce(
+        {
+          brand: Far('otherBrand', {
+            getAllegedName: () => 'somename',
+            isMyIssuer: async () => false,
+            getDisplayInfo: () => ({}),
+          }),
+          value: 4n,
+        },
+        mockBrand,
       ),
     {
-      message: /The brand in the allegedAmount .* in 'coerce' didn't match the amountMath brand/,
+      message: /The brand in the allegedAmount .* in 'coerce' didn't match the specified brand/,
     },
     `coerce can't take the wrong brand`,
   );
   t.throws(
-    () => coerce(3),
-    { message: /The brand in allegedAmount .* is undefined/ },
+    () => m.coerce(3n, mockBrand),
+    {
+      message: `The brand in amount (a bigint) doesn't look like a brand. Did you pass a value rather than an amount?`,
+    },
     `coerce needs a brand`,
   );
+});
 
-  // getValue
-  t.is(getValue(make(4)), 4n);
+test('natMathHelpers coerce no brand', t => {
+  t.throws(
+    // @ts-ignore
+    () => m.coerce(m.make(4n, mockBrand)),
+    {
+      message: "The brand (an undefined) doesn't look like a brand.",
+    },
+    `brand is required in coerce`,
```

Comment:
The error message complains about the malformed brand, without mentioning that what it's unhappy about is the missing brand in the first arg, which is supposed to be an amount. This was confusing because the second parameter *is* a brand and is supposed to be a brand. Any chance this message can be improved?

```suggestion
    `amount is required in coerce`,
```

Reply:
I think this is misreading the test, which I did as well rereading this. There's a `m.make(4n, mockBrand)` which produces a full amount, but it actually the second argument, the `brand`, that is missing so this message is correct.